src/components/elements/auto-carousel.js

```diff
@@ -0,0 +1,91 @@
+import React from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import Container from 'components/containers/container'
+
+const CarouselContainer = styled(Container)`
+    overflow: hidden;
+    justify-content: space-between;
+
+    div {
+        transition: transform ${props => (props.transition ? '1s' : '0')};
+        transform: translate(${props => props.position}rem);
+    }
+`
+const CarouselWrapper = styled.div`
+    max-width: 9.6rem;
+    margin-right: 3.2rem;
+`
+const Icons = props =>
+    props.iconsArray.map(Element => (
+        <CarouselWrapper key={Element.key}>
+            <Element.Component />
+        </CarouselWrapper>
+    ))
+
+class AutoCarousel extends React.Component {
+    intervalRef = undefined
+    timeoutRef = undefined
+    MyRef = React.createRef()
```

Comment:
please change all the non functions to snake case also

Reply:
Done